react-native-pokedex/src/components/card.jsx

```diff
@@ -0,0 +1,201 @@
+import React from "react";
+import { View, Text, StyleSheet, Image } from "react-native";
+import { useFonts } from "expo-font";
+
+
+const Card = (props) => {
+  const [loaded] = useFonts({
+    Montserrat: require('./assets/fonts/Montserrat.ttf'),
+  });
+
+  if (!loaded) {
+    return null;
+  }
+
+  const setStyles = (type) => {
+    return styles[type]
+  }
+
+  return (
+    <View style={styles.card, setStyles(props.type)}>
+      <View style={styles.container}>
+        <Text style={styles.text}>props.pokemon</Text>
+        <View style={styles.container}>
+        <Image style={styles.sprite} source={props.sprite}/>
```

Comment:
I couldnt get a sprite to render, to test why, i tried setting the source to 
source={{ uri: props.sprite, width: 100, height: 100 }} and that seemed to work

Reply:
Couldn't get this to work, set up how I got mine to work. Can you show me an example of how you got that to work? :)